programs/nodeos/logging.json

```diff
@@ -15,7 +15,8 @@
             "level": "error",
             "color": "red"
           }
-        ]
+        ],
+        "flush": true
```

Comment:
Is there any difference by setting flush to be true or false here?

Reply:
Not very sure what will happen if set flush to false. However, as this PR is merge back, it should be synchronized with original PR.

Reply:
I wasn't sure about flush's value because the change description "Add flush option to example logging.json since someone might want to set it to false for stdout logging." I am going to approve this PR as  it's a back-merge.